src/hotspot/share/services/diagnosticCommand.hpp

```diff
@@ -577,6 +577,27 @@ class CompileQueueDCmd : public DCmd {
   virtual void execute(DCmdSource source, TRAPS);
 };
 
+class PerfMapDCmd : public DCmd {
+public:
+  PerfMapDCmd(outputStream* output, bool heap) : DCmd(output, heap) {}
+  static const char* name() {
+    return "Compiler.perfmap";
+  }
+  static const char* description() {
+    return "Write map file for Linux perf tool.";
+  }
+  static const char* impact() {
```

Comment:
This operation does not seems to happen in Safepoint. It can be defined as "Low" IMHO.